integrations/matplotlib/notebooks/Neptune_Matplotlib_Integration.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Neptune-Matplotlib-Integration.ipynb",
```

Comment:
@SiddhantSadangi could you please change the name of the file to this as well. 
Strangely the name of the file is Neptune_Matplotlib_Integration.ipynb while here we have Neptune-Matplotlib-Integration.ipynb

Also, I think we should change Integration to Support right?